Makefile

```diff
@@ -4,22 +4,40 @@ default: build
 build:
 	docker build -t rpi-homebridge-dev .
 
+clean:
+	-docker kill homebridge
+	-docker rm homebridge
+
 # Target to build and run and subsequently remove image
 run: build
 	docker run --net=host --rm \
-		-p "localhost:51826:51826"
+		-p "51826:51826" \
 		-v "$(shell pwd)/config.json:/root/.homebridge/config.json" \
 		-v "$(shell pwd)/plugins.txt:/root/.homebridge/plugins.txt" \
+		-v "$(shell pwd)/persist:/root/.homebridge/persist" \
```

Comment:
Is this a standard directory? You could also persist the entire `/root/.homebridge` directory, as it is a volume.

Reply:
That works! Thanks!

Reply:
That works. Thanks!